Gopkg.lock

```diff
@@ -22,7 +22,7 @@
   version = "v19.0.0"
```

Comment:
Let's upgrade to v20.1.0

Reply:
I'd like to do that as well--many services (including containerservices itself) have been updated in the SDK since v19.0.0. I think it would be safest to use the go-autorest version that was actually used to generate the Go SDK, if we can find that out.

Reply:
Apparently that was [10.5.3](https://github.com/Azure/azure-sdk-for-go/blob/v20.1.0/Gopkg.toml#L25). Close enough.

Reply:
I checked with Martin Strobel earlier, the go sdk maintainer. He says using 10.15.4 of go-autorest along with v20.1.0 of go sdk should be fine.

Reply:
OTOH, upgrading sdk-for-go has a chance of breaking some API acs-engine is already using (and this has a cascade effect when vendored into AKS). If this is passing, maybe that should be a separate PR?

Reply:
Sure!